neo/rawio/baserawio.py

```diff
@@ -723,6 +783,7 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
         by channel_indexes (local index inner stream).
 
         All channels indexed must have the same size and t_start.
+        :TODO: won't this be the case now if they are all part of the same stream?
```

Comment:
Yes. You can remove this 786 line.
It was for the old story.
thank you.

Reply:
Done.